readme.md

```diff
@@ -25,3 +34,6 @@ docker build -t deepnote/ir:<TAG>  --build-arg R_BASE_VERSION=<some_version> ./i
 The image is based on tensorflow docker image and adds packages that are typical part of our other images (`git` most importantly).
 
 We follow the tagging of `tensorflow` image. So `tensorflow/tensorflow:2.4.1-gpu` becomes `deepnote/tensorflow:2.4.1-gpu`.
+
+## Continuous integration
```

Comment:
I see now, I previously didn't realise that's what the https://github.com/deepnote/environments/tree/main/python/hooks are for 🪄